tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -372,12 +372,8 @@ public function getETag(string $user, string $spaceName, string $fileName): stri
 	 * @return void
 	 */
 	public function setLastShareData(): void {
```

Comment:
remove this method and use `$this->featureContext->addToCreatedPublicShares()` instead